helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/listener/HelpDeskRoomEventListener.java

```diff
@@ -115,8 +132,30 @@ public void onSymUserLeftRoom(SymUserLeftRoom symUserLeftRoom) {
     SymUser symUser = symUserLeftRoom.getAffectedUser();
     SymStream symStream = symUserLeftRoom.getStream();
 
-    if (isBotUser(symUser) && isAgentStreamId(symStream)) {
-      LOGGER.warn("Bot user was removed from the agent room");
+    if (isAgentStreamId(symStream)) {
+      if (isBotUser(symUser)) {
+        LOGGER.warn("Bot user was removed from the agent room");
+      }
+    } else {
+      if (symStream.getMembers().size() <= 1) {
+
+        LOGGER.info("Only the bot was left in the ticket room. Reopening ticket in the Agent room");
+        // Update ticket first
+        Ticket ticket = ticketClient.getTicketByServiceStreamId(symStream.getStreamId());
+        ticket.setAgent(null);
+        ticket.setState(TicketClient.TicketStateType.UNSERVICED.toString());
+        ticketClient.updateTicket(ticket);
+
+        // Resend ticket message to the agent room
+        SymMessage symMessage = SymMessageBuilder.message("").build();
+        symMessage.setFromUserId(symUser.getId());
```

Comment:
You should set the bot user id as the message sender.

```
SymUser localUser = symphonyClient.getLocalUser();
symMessage.setFromUserId(localUser.getId());
```

Reply:
Fix'd